ci/cloudbuild/dockerfiles/demo-centos-7.Dockerfile

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG DISTRO_VERSION=7
```

Comment:
Thanks for cleaning up these `DISTRO_VERSION`s. I originally copied docker files over as-is, but planned to clean this up later. Apparently I forgot, and even forgot to file an issue so I don't forget ;-)